wd_helper.au3

```diff
@@ -1027,6 +1033,25 @@ Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
 	If $iErr = $_WD_ERROR_Success Then
 		If $sNodeName = 'select' Then ; check if designated element is <select> element
 			Switch $sCommand
+				Case 'deselectAll'
+					$sScript = "return arguments[0].selectedIndex = -1"
+					$vResult = _WD_ExecuteScript($sSession, $sScript, __WD_JsonElement($sSelectElement), Default, $_WD_JSON_Value)
+					$iErr = @error
+
+				Case 'multiSelect' ; https://stackoverflow.com/a/1296068/5314940
+					$sScript = _
+							"var LabelsToSelect = ['" & _ArrayToString($vParameters, "', '") & "'];" & _
```

Comment:
Have you tested to see what happens here if $vParameters isn't an array?

Reply:
As so far no.
Thanks to pointing this.
Will do test.

Reply:
I need more time for this testing (I hope will handle this in comming weekend)

Reply:
Currently there is validation:
```auotit
	If UBound($aParameters, $UBOUND_ROWS) = 0 Or UBound($aParameters, $UBOUND_COLUMNS) <> 0 Then
		$iErr = $_WD_ERROR_InvalidArgue
		$iExt = 41 ; $iExt from 41 to 49 are related to _WD_ElementSelectAction()
	Else
		$sScript = _
				"var LabelsToSelect = ['" & _ArrayToString($aParameters, "', '") & "'];" & _
```